client/viame-web-common/components/AttributeInput.vue

```diff
@@ -50,7 +50,12 @@ export default defineComponent({
     });
 
     function blurType(e: KeyboardEvent) {
-      (e.target as HTMLInputElement).blur();
+      const target = (e.target as HTMLInputElement);
+      // Datalist needs to reset if we blur on no input
+      if (props.values && props.values.length && target.value === '') {
```

Comment:
target.value will be `' '` not `''` for datalist index 0.  Does this capture that case?

Reply:
Selecting that manually from the datalist in the dropdown immediately sets it to undefined and bypasses this function.  This is for only hitting enter after typing, (or specifically not typing anything).  Now hitting enter without adding anything will return it to the default initial value (Same as the type dropdown).  Adding in only spaces will remove the value (same as datalist index 0).

With the current setup if you select the 0 index item it immediately executes the `change`, emitting undefined which will hide the data value if you are editing a single attribute.  So the dropdown bypasses the `blurType` function.

These discrepancies and confusion are why I added that other issue about the behavior of the Type dropdown in `TrackListItem`.